src/core/files.py

```diff
@@ -358,6 +358,33 @@ def serve_file_to_browser(file_path, file_to_serve, public=False):
     return response
 
 
+def serve_pdf_galley_to_browser(request, file, article):
+    """
+    Serves a file to the browser so that it displays in the browser.
+    :param request: HttpRequest object
+    :param file: File object
+    :param article: Article object
+    :return: HttpResponse
+    """
+    file_path = os.path.join(
+        settings.BASE_DIR,
+        'files',
+        'articles',
+        str(article.id),
+        str(file.uuid_filename)
+    )
+
+    try:
+        response = HttpResponse(
+            FileWrapper(open(file_path, 'rb'), 8192),
```

Comment:
Why hardcode he buffer size to 8192 here? Its the default behaviour of `FileWrapper`. if anythong we should ask the os via `os.stat()` or use the default `io.DEFAULT_BUFFER_SIZE` (which is what we are doing on the `open(file_path, 'rb')` call)